### ISSUE TITLE
[Bug Report][3.1.10] Can't use menu with keyboard in app-bar

### ISSUE THREAD

**Issue by dennismcknight-tr** [vuetifyjs/vuetify#16940]
### Environment
**Vuetify Version:** 3.1.10
**Vue Version:** 3.2.47
**Browsers:** Chrome 111.0.0.0
**OS:** Windows 10

### Steps to reproduce
Include a `v-menu` inside of a `v-app-bar`.

### Expected Behavior
You should be able to configure a `v-menu` inside a `v-app-bar` and both activate it AND jump into it immediately via the keyboard.

### Actual Behavior
Either the menu doesn't display at all (likely offscreen or in an otherwise invisible location) OR it displays but you must tab through the rest of the page first before it circles back to the menu dropdown.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqtVE1v2zAM/SuEeugG1HaTBhiQtkMG7FAMuw471D0otpJq0RckOlsW5L+Psp3EH+kOw3KKyMfHxydaz3v2ybl0Wwk2Zw8otFMcxcfcADxsE+5c/fd4SJbcQylMkLh7zFlhteMF5gyEEluO0hqKTulcKB4C/Y81zkvN/S5BaxXV56ylrEnbYIISVdO2+T0JpSx8t16VJ3T2Fpx4AgkRPuuGtDAVcERevJKSq3h8sqoUXQUEPM4MVzSKpCmsJ/genLcuwKEHrnmXaLoRgMKqumapKpGzfo7g0pSUrOkGZERXyu0gBPDFCvgWSGcfmg2x0RAS0x0m693gSbKSAYdjxFgiCQ+LQsliQxrvx/o6uMZ0CLhTgsArazAJ8reYT2bu16gS4Ktd2wqHQ4wIRyOd8r3BBkPEQLzR43rSsd3PNnJaW83l2aLoN8h4H7116HpbQ477q3kyue1vLDl+2bEcP1da72DS1di9nwvF50YqmV3gmv5HrrvLXP29qn09O1a/A423FOmtFx1D4aVDCAKrmJbaWY+wBy9WcICVtxqu6WW5juDCmoBQSq7sGh4j5N2KqyDe30fehok42A1raEiFS38Ea+hd2kcZeZugj2gOdSTGiD6ec/aK6MI8yyrjNuuUXqZsQbnMVwalFklp9eIunaazDzRuwG48FUEnS29/0idHHXN20yFHudq91aBNE+8kndw2vG0sUXwZIvGIMKMuW+ETLwztXty+v6gfYHsTDHKjKWLPQ24O5KiSZhMGZhahNvL5KO0fxqspYv1LbHO4YY08atiUsJc/AeDtbg==)

### Other comments
Visit the reproduction link.  When clicking the Joe User button in the app-bar I cannot get the dropdown menu to render correctly AND be used by the keyboard.  It either doesn't appear correctly at all OR, when it does display, you have to tab through the rest of the page first before it loops back around.

In the documentation for menus it says that in order for keyboard usage to work you have to specify an `attach` attribute to the menu, however when I use this attribute the overlay and dropdown don't appear sized or stacked correctly and it becomes invisible. If this isn't a bug I would appreciate an example of how this is supposed to work.  

I've had better luck using `attach` on `v-menu`s outside of a `v-app-bar` as it seems to work correctly there.
